src/wiki/development/build-instructions.md

```diff
@@ -123,15 +123,15 @@ flatpak-builder --user --install flatbuild org.prismlauncher.PrismLauncher.yml
 2. Choose a place to install Qt.
 3. Choose the components that you wish install.
 
-   - You need Qt 5.12.x 64-bit ticked. (or a newer version)
-      - Alternatively you can choose Qt 6.0 or newer
-   - You need Tools/Qt Creator ticked.
-   - Other components are selected by default, you can un-tick them if you don't need them.
+    - You need Qt 5.12.x 64-bit ticked. (or a newer version)
+    - Alternatively you can choose Qt 6.0 or newer
```

Comment:
the extra indentation here got removed
```
        - Alternatively you can choose Qt 6.0 or newer
```
its already merged now, so guess needs a new pr

Reply:
Oh, whoops.